bundles/framework/divmanazer/component/SelectList.js

```diff
@@ -182,6 +182,8 @@ Oskari.clazz.define('Oskari.userinterface.component.SelectList', function (id) {
         }
         if (!supressEvent) {
             this.resetToPlaceholder();
+        } else {
+            this.element.find('select').trigger('chosen:updated');
```

Comment:
Should suppressEvent be renamed to suppressChangeEvent? That's the event that is not triggered if the reset is called with parameter true?